tests/test_listening_history.py

```diff
@@ -18,18 +19,26 @@ def setUp(self):
   def tearDown(self):
     super(ListeningHistoryTestCase, self).tearDown()
     ListeningHistory.query.delete()
-    for e in self.changed_episodes:
-      e.real_duration = None
+    episodes = \
+      Episode.query.filter(Episode.id.in_(self.changed_episodes.keys())).all()
+    for episode in episodes:
```

Comment:
If you are only grabbing id, why are you serializing entire episodes? Can you possibly in the query return just the ids?

Reply:
oh these are tests... awks... :)